Source/santad/BUILD

```diff
@@ -213,3 +213,25 @@ santa_unit_test(
     ],
 
 )
+
+santa_unit_test(
+    name = "SNTApplicationTest",
+    data = [
```

Comment:
s/data/resources

Reply:
I'd actually prefer to keep it as data instead of a resource - the filegroup rule is easier to work with for non-Apple rules (such as any integration tests in the future).